lib/geckoboard/dataset.rb

```diff
@@ -16,6 +16,10 @@ def put(data)
       client.put_data(id, payload_formatter.format(data))
     end
 
+    def post(data, options={})
```

Comment:
@boxofrad i'm not sure about shovelling. Would you see that as requiring an explicit commit? Or would each shovel trigger a request?

Also it might suggest that other operators could be used that alter the dataset (something we have no plans to support AFAIK)